prime-router/metadata/hl7_mapping/ORU_R01/RADx_MARS/RADx_MARS-observation-result.yml

```diff
@@ -0,0 +1,14 @@
+constants:
+  hl7ObservationPath: '/PATIENT_RESULT/ORDER_OBSERVATION(%{orderIndex})/OBSERVATION(%{resultIndex})'
+  symptomaticAoe: '%service.supportingInfo.resolve().where(code.coding.code = %symptomaticAoeLoinc)'
+elements:
+
+  # RADx MARS requirement:
+  # Only set the illness onset date if symptomatic AOE answer is Yes
```

Comment:
Just calling out that the spec specifically says to only set the date if it matches the format `{YYYYMMDDHHMMSS[+/-ZZZZ]}` and example bundle has it as `{YYYYMMDD}`